clusterloader2/pkg/measurement/common/simple/scheduling_throughput.go

```diff
@@ -135,23 +135,23 @@ func (s *schedulingThroughputMeasurement) gather() ([]measurement.Summary, error
 	s.stop()
 	klog.Infof("%s: gathering data", s)
 
-	summary := &schedulingThroughput{}
+	st := &schedulingThroughput{}
```

Comment:
I actually prefer summary - st isn't very reader-friendly

Reply:
Renamed to "throughputSummary"